Getting instance ElementMetadataKeyValuePairType instead creating new.

vmalyk opened the pull request:
Solves #924.

vmalyk committed:
getting instance ElementMetadataKeyValuePairType instead creating new.

klickparkdominik commented:
I am the author of the referenced issue #924 . I tested this PR in my environment and can confirm that it solves the described problem.

vmalyk commented:
hi @wisconaut , @mcop1 ,

any feedback regarding PR and if it's required some adjustments - i'll be glad getting feedback and fix?

Thank you

kingjia90 commented:
LGTM but could it be merged as bugfix, right? Just wondering, as i've noticed that you switched base branch to 1.x, or am i missing something? TIA

vmalyk commented:
@kingjia90 , 

yeah, I've changed, because I checked merged PRs and it was like fixes, but to 1.x. And I thought might be it's reason why PR is wrongly provided from my side.
I've updated PR and back 1.9 for merging.
Can you have a look?
